### PR TITLE
Adding Certificate Instruction Page

### LINKED ISSUE #293: Put instructions for certificates
We have much blank text on different pages such as:
- domains screen description
- domain creation/editing 
- certificate (before it was created and after)

All these should be replaced with the meaningful text that will help user to figure out what is going on

### PR DESCRIPTION
**Pull request by Ririio**
This populates the route `/certificate/instructions` with actual information

Closes #293

### CODE REVIEW

**Comment by Ririio** [comment]
> Lastly, you can check the status of your SSL certificate by visiting [SSL Shopper.](https://www.sslshopper.com/ssl-checker.html)

I may have worded it wrong, but it was meant for the user to use to check if their domain has valid certificate

> Let's Encrypt certificates are valid for [90 days](https://letsencrypt.org/docs/faq/#:~:text=Our%20certificates%20are%20valid%20for,your%20certificates%20every%2060%20days.).

I'm a bit confused, but I'm referring here for the "expiration" date of the user's certificate, I recall the ITS people are the one who establishes this

> Can you use "DNS Records" like we do everywhere else? If you also want to link to that page, that's good too.

Sure I'll refer to our page

**Comment by humphd** [comment]
> > Lastly, you can check the status of your SSL certificate by visiting [SSL Shopper.](https://www.sslshopper.com/ssl-checker.html)
> 
> I may have worded it wrong, but it was meant for the user to use to check if their domain has valid certificate

This isn't clear from what you wrote.

> > Let's Encrypt certificates are valid for [90 days](https://letsencrypt.org/docs/faq/#:~:text=Our%20certificates%20are%20valid%20for,your%20certificates%20every%2060%20days.).
> 
> I'm a bit confused, but I'm referring here for the "expiration" date of the user's certificate, I recall the ITS people are the one who establishes this

ITS has nothing to do with this.  It's a function of the length of the certificate that Let's Encrypt grants.

Maybe you're thinking of how long the domains last?  That is arbitrary.

File: app/components/certificate/description.tsx [hunk 1]
```diff
@@ -16,10 +27,21 @@ export default function DescriptionSection({
...
   return (
-    <Flex flexDirection="column" gap="3" marginTop={{ base: 1, md: 5 }} fontSize="md">
-      <Text maxW={600}>{description}</Text>
+    <Flex flexDirection="column" gap="3" fontSize="md">
+      <Text maxW={600}>
+        {description}
+        {link && (
+          <>
+            &nbsp;To learn more, refer to this&nbsp;
+            <Link as={RemixLink} to={link}>
```
Comment (Genne23v) [inline app/components/certificate/description.tsx:39]: It's not a big deal, but I think it's good to align with other page sentence. `To learn more, refer to our information page`

File: app/routes/__index/certificate/information.tsx [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+          <br />
+          <br />
+          Unfortunately, there are malicious websites that do not provide any sense of security and
+          may not be able to prove their identity to your web browser, potentially leading to safety
+          issues when trying to access them. To ensure your safety while browsing the internet,
+          user-friendly browsers like Chrome automatically block access to such sites that could
+          harm your computer
+          <br />
+          <br />
+          Unfortunately, there are malicious websites that do not provide any sense of security and
```
Comment (Genne23v) [inline app/routes/__index/certificate/information.tsx:64]: This paragraph is repeated.

File: app/routes/__index/certificate/information.tsx [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+              depends on their rules and regulations. They'll be able to provide you with the
+              necessary information in this regard. You can contact them by clicking&nbsp;
+              <Link href="https://students.senecacollege.ca/spaces/190/support/wiki/view/1473/contact-its">
+                here
+              </Link>
+            </Text>
+          </FaqAccordion>
+          <FaqAccordion title="Can I use a single SSL Certificate for multiple domains?">
+            The certificate that has been issued covers all the domains you've set up under
+            dns-records.
```
Comment (Genne23v) [inline app/routes/__index/certificate/information.tsx]: Should it be `DNS records`? Perhaps, `in DNS record table` is clearer?

**Review by Genne23v — CHANGES_REQUESTED** [review]

**Review by humphd — CHANGES_REQUESTED** [review]
Please do the same width treatment to the dns info page, so they match.

**Review by humphd — CHANGES_REQUESTED** [review]

File: app/routes/__index/certificate/information.tsx (hunk 3 above)
Comment (Ririio) [reply app/routes/__index/certificate/information.tsx]: sure I'll refer to both the instruction and the table

**Review by dadolhay — APPROVED** [review]

**Pull request merged**